Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantAdjustAttribute.cs

```diff
@@ -15,6 +15,18 @@ public abstract partial class PlantAdjustAttribute : ReagentEffect
         [DataField]
         public float Prob { get; protected set; } = 1; // = (80);
 
+        /// <summary>
+        /// Localisation key for the name of the adjusted attribute. Used for guidebook description.
+        /// </summary>
+        [DataField]
+        public string Attribute { get; protected set; } = "missing";
```

Comment:
mark this `required`, don't give it a default. Also, rename to something more descriptive. `GuidebookAttribute`?